helm_deploy/prisoner-content-hub-backend/templates/ingress-jsonapi.yaml

```diff
@@ -3,15 +3,17 @@
 apiVersion: networking.k8s.io/v1beta1
 kind: Ingress
 metadata:
-  name: {{ $fullName }}-jsonapi
+  name: {{ $fullName }}-jsonapi-ingress
```

Comment:
Have to use a different name from previous ingress controllers.  (Although what the name is doesn't actually matter).